Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,29 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const result =
+    // step 1.2.4
+    Math.floor(
+      100 *
+        tasks
+          // step 1.2.1
+          .map(x => x.duration / 60)
+          // step 1.2.2
+          .filter(x => x >= 2)
+          // step 1.2.3
+          .map(x => x * hourlyRate)
+          .reduce((accumulator, number) => accumulator + number)
+    ) / 100;
+  return result;
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
+console.log(earnings);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
 
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+console.log(`Maartje has earned €${earnings}`);
```

Comment:
Here, at presentation, is the correct point to round of the earnings. Rather than using `Math.round()` you can simply use `Number.toFixed(2)`:

```js
console.log(`Maartje has earned €${earnings.toFixed(2)}`);
```